### PR TITLE
feat: Set log level of a specific package using logging trait

### PR DESCRIPTION
**Pull request by realMartinez**
- added capability to the logging trait to set log level of user specified package
- added unit test

_Usage:_
` -t "logging.category.'org.test'=DEBUG"`

**Release Note**
```release-note
feat: Set log level of a specific package using logging trait
```

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Review by squakez — COMMENTED** [review]
You need to `make generate` in order to generate the trait documentation and CRD associated with the change. Thanks.

File: pkg/trait/logging_test.go [hunk 1]
```diff
@@ -212,3 +213,31 @@ func TestJsonLoggingTrait(t *testing.T) {
 	assert.True(t, logFormatIsNotDefault)
 	assert.NotEmpty(t, env.ExecutedTraits)
 }
+
+func TestLoggingCategory(t *testing.T) {
```
Comment (squakez) [inline pkg/trait/logging_test.go:217]: Make another test to include 2 variables and make sure the 2 different packages are correctly transformed.

**Comment by realMartinez** [comment]
I generated the docs and also made the changes you requested.

**Review by squakez — APPROVED** [review]
Great, good job! One more thing. We may provide an example inline with the new parameter as we do for route annotations.

**Comment by realMartinez** [comment]
> Great, good job! One more thing. We may provide an example inline with the new parameter as we do for route annotations.

I amended the documentation to include CLI usage example

**Review by gansheer — APPROVED** [review]

**Comment by squakez** [comment]
@realMartinez I'm converting to a draft as I understand you're going to work on E2E test to validate the feature. Thanks.

**Comment by realMartinez** [comment]
Need to close this PR, will link this to the new one.

**Pull request closed without merging**